---
title: "Laboratory report" 
editor_options: 
  chunk_output_type: console
---

## Introduction

Analysis of gene expression through fluorescence-based real-time quantitative polymerase chain reaction (qPCR) is a well-established practice used in a variety of exercise studies [@kuang2018]. This method quantifies target gene expression in biological samples, such as from blood or muscle tissue. The qPCR analysis is widely used and there are numerous protocols and methods for conducting the analysis. In our study we used the SYBR Green method where a fluorescent dye binds to the DNA during amplification. This fluorescence allows real-time tracking of the reaction, as the signal intensity correlates with the amount of DNA present [@kuang2018]. The fluorescence reaches a pre-determined threshold, referred to as the cycle threshold (CT), which reflects the level of gene expression. A lower CT indicates higher gene expression, as fewer amplification cycles were required to cross the threshold [@Livak].

In exercise physiology, qPCR is frequently used to investigate changes in gene expression that occur with training. In this study we aim to check primer efficiency and perform targeted amplification of cDNA using specific primers. Additionally, we aim to find out how strength training affects the gene expression of different muscle fiber types as participants underwent a two-week strength training program. The different muscle fiber types we looked at were type I, type IIa and type IIx. Type I fibers, slow-twitch, are highly oxidative and can sustain contraction over long periods without fatigue [@Pette2000]. Type IIa fibers utilize both aerobic and anaerobic metabolic pathways, have intermediate resistance to fatigue and produce more force than type I fibers [@Pette2000]. Lastly, type IIx fibers rely predominantly on anaerobic metabolism and are specialized for rapid and powerful contractions [@Pette2000].

## Materials

-   A real-time PCR machine (We use QuantStudio 5)

-   A qPCR reaction plate

-   Nuclease-free water and pipette tips

-   SYBR-green Master mix

## Method

Prior to the experiment, the lab manager prepared the cDNA, which was extracted from samples collected during a study where participants underwent a two-week strength training program.

First, we created a dilution series to test the primers (@tbl-dilution). We moved 2µl of the sample from tube 1 to tube 2a, and 10µl from 1 to 2b, then vortexed tube 2a and 2b so that the sample and water (H~2~O) would mix. Next, we moved 2µl from 2a to 3a and 2µl from 2b to 3b, then vortexed tube 3a and 3b. Lastly, we moved 2µl from 3a to 4a and 2µ from 3b to 4b, then vortexed tube 4a and 4b.

```{r}
#| message: false
#| echo: false
#| warning: false
#| label: tbl-dilution 
#| tbl-cap: "Dilution series"

library(gt)
library(tidyverse)

table_data <- data.frame(
  row.names = c("Fortynning", "prøve", "H2O"),
  `1` = c("1", "30µl", "0µl"),
  `2a` = c("1/10", "2µl", "18µl"),
  `3a` = c("1/100", "2µl", "18µl"),
  `4a` = c("1/1000", "2µl", "18µl"),
  `2b` = c("1/2", "10µl", "10µl"),
  `3b` = c("1/20", "2µl", "18µl"),
  `4b` = c("1/200", "2µl", "18µl")
)


colnames(table_data) <- c("1", "2a", "3a", "4a", "2b", "3b", "4b")

table_data %>% gt() %>%
  tab_options(
    table.font.size = px(9), 
    table.width = px(500)    
  )



```

We then combined a master mix consisting of 250µl SYBR-green, 50µl primer mix (MHC1, MHC2a, MHC2x, or MHCb2m), and 100µl H~2~O. Subsequently, we loaded the plate with primer-specific master-mix following the outline of our pipetting scheme (@tbl-pipette). 8µl of the master mix was added to the wells along with 2µl of cDNA sample.

```{r}
#| message: false
#| echo: false
#| warning: false
#| label: tbl-pipette 
#| tbl-cap: "Pipetting scheme"

library(tidyverse)
library(gt)

table_data1 <- data.frame(
  Row = c(NA, "A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L"),
  "13" = c("Fp1", "myhc 1", "myhc 1", "myhc 1", "myhc 2a", "myhc 2a", "myhc 2a", "myhc 2x", "myhc 2x", "myhc 2x", "myhc", "myhc", "myhc"),
  "14" = c("Fp2", "myhc 1", "myhc 1", "myhc 1", "myhc 2a", "myhc 2a", "myhc 2a", "myhc 2x", "myhc 2x", "myhc 2x", "myhc", "myhc", "myhc"),
  "15" = rep(NA, 13),
  "16" = rep(NA, 13),
  "17" = c(NA, "cmyc 1", "cmyc 1", "cmyc 1", NA, NA, NA, NA, NA, NA, "cmyc 1", "cmyc 1", "cmyc 1"),
  "18" = c(NA, "cmyc 2a", "cmyc 2a", "cmyc 2a", NA, NA, NA, NA, NA, NA, "cmyc 2a", "cmyc 2a", "cmyc 2a"),
  "19" = c(NA, "cmyc 3a", "cmyc 3a", "cmyc 3a", NA, NA, NA, NA, NA, NA, "cmyc 3a", "cmyc 3a", "cmyc 3a"),
  "20" = c(NA, "cmyc 4a", "cmyc 4a", "cmyc 4a", NA, NA, NA, NA, NA, NA, "cmyc 4a", "cmyc 4a", "cmyc 4a"),
  "21" = c(NA, "cmyc 2b", "cmyc 2b", "cmyc 2b", NA, NA, NA, NA, NA, NA, "cmyc 2b", "cmyc 2b", "cmyc 2b"),
  "22" = c(NA, "cmyc 3b", "cmyc 3b", "cmyc 3b", NA, NA, NA, NA, NA, NA, "cmyc 3b", "cmyc 3b", "cmyc 3b"),
  "23" = c(NA, "cmyc 4b", "cmyc 4b", "cmyc 4b", NA, NA, NA, NA, NA, NA, "cmyc 4b", "cmyc 4b", "cmyc 4b")
) 

colnames(table_data1) <- c("row", "13", "14", "15", "16", "17", "18", "19", "20", "21", "22", "23") 

table_data1[is.na(table_data1)] <- ""

table_data1 %>% gt() %>%
  tab_options(
    table.font.size = px(9), 
    table.width = px(500)    
  )



```

The plate was then covered with plastic and centrifuged at 1200rpm for 1 minute. The PCR samples were analyzed using real-time PCR and QuantStudio software. The PCR process consisted of 40 cycles. After the PCR process was completed, we extracted the results in the form of CT values.

### Data Analysis

We analyzed our data in excel.

#### Dilution Series

Here we calculated the average CT value for the triplicates as well as the base-10 logarithm of the sample quantity (Log(sample.quan). The slope was then calculated by plotting Log(sample.quan) against the average CT. A linear regression was performed to find the best-fit line, and the slope was derived. Primer efficiency was then calculated from the slope. Lastly, we calculated the standard deviation (SD) among replicates and the coefficient of variation (CV).

#### Gene Expression

We processed our myosin heavy chain gene expression data by using the Delta-Delta-Ct (ΔΔCt) method. Cycle threshold (Ct) values for both the target gene and the reference gene (b2m) were measured. We calculated the average for multiple technical replicates to reduce variability. We then calculated ΔCt which quantifies the expression of the target gene relative to the reference gene within the sample. Lastly, we calculated the relative expression level (2\^-ΔΔCt) representing the fold change in expression of the target gene in the sample relative to the calibrator.

## Results

### Dilution Series

```{r}
#| message: false
#| echo: false
#| warning: false
#| label: tbl-x 
#| tbl-cap: "Dilution series results"

table_data2 <- data.frame(
  Dilution = c("1", "1/2", "1/10", "1/20", "1/200"),
  Ct1 = c(28.678, 29.414, 31.776, 33.241, "Undet."),
  Ct2 = c(28.708, 29.620, 31.416, 32.653, "Undet."),
  Ct3 = c(29.155, 29.264, 32.413, "Undet.", 34.574),
  `Avg Ct` = c(28.847, 29.433, 31.868, 32.947, 34.574),
  `Sample quan` = c(1, 0.5, 0.1, 0.05, 0.005),
  `Log(sample quan)` = c(0, -0.301, -1, -1.301, -2.301),
  `Slope` = c(-2.6104, NA, NA, NA, NA),
  `PE (%)` = c(141.5901, NA, NA, NA, NA))  

table_data2[is.na(table_data2)] <- ""


table_data2 %>% 
  gt() %>%
  tab_source_note(
    source_note = "A slope of -2.6 indicates that the primer efficiency is not optimal as it is calculated to be 142%. We can see that our observation is not linear") %>%
  tab_options(
    table.font.size = px(9), 
    table.width = px(500)    
  )


```

```{r}
#| message: false
#| echo: false
#| warning: false
#| label: tbl-x1
#| tbl-cap: "Average Ct values, standard deviations (SD), and coefficients of variation (CV) for pooled qPCR samples at three dilution levels"

library(tidyverse)
library(gt)


pooled_data <- data.frame(
  `Pooled sam` = c(1, 0.5, 0.1),
  Avg = c(28.847, 29.433, 31.868),
  SD = c(0.26704902, 0.17887641, 0.50500096),
  `CV` = c("0.93 %", "0.61 %", "1.58 %"))


pooled_data %>% 
  gt() %>% tab_source_note(
    source_note = "The standard deviations (SD) are relatively low for the dilutions in the pooled sample, indicating precision in the results. The coefficient of variation (CV) is lowest for the 1/2 diluted sample at 0.61% and highest for the 1/10 diluted sample at 1.58%. This suggests that there is greater variability between the measurements for the most diluted samples. Additionally, the CV of the undiluted sample is higher than that of the 1/2 diluted sample.") %>%
  tab_options(
    table.font.size = px(9), 
    table.width = px(500)    
  )

```

```{r}
#| echo: false 
#| message: false
#| warning: false
#| label: fig-dilution-series 
#| fig-cap: "Relationship between the average cycle threshold values and the logarithm of sample quantities in our dilution series"

library(ggplot2)

data <- data.frame(
  Fortynning = c("1", "1/2", "1/10", "1/20", "1/200"),
  Avg_Ct = c(28.847, 29.433, 31.868, 32.947, 34.574),
  Log_sample_quan = c(0, -0.301, -1, -1.301, -2.301))

ggplot(data, aes(x = Avg_Ct, y = Log_sample_quan, color = Fortynning)) +
  geom_point(size = 2) +  
  geom_smooth(method = "lm", se = FALSE, color = "black", size = 0.5, linetype = "dotted") +  
  labs(
    x = "Average Ct",
    y = "Log(sample quan)",
    caption = "The plot shows a negative linear relationship between average Ct values and log (sample quantity). The dotted \nblack line represents a linear regression fit to the data, emphasizing the inverse proportionality. \nDifferent colors represent the dilution level.") +
  theme_minimal() +
  scale_color_manual(values = c("royalblue3", "paleturquoise", "palegreen2", "wheat4", "pink")) +
  theme(legend.title = element_blank(), 
    plot.title = element_text(hjust = 0.5),
    legend.position = "top",
    plot.caption = element_text(hjust = 0, size = 7, face = "italic"))


```

### Gene Expression


```{r}
#| message: false
#| echo: false
#| warning: false
#| label: tbl-ct 
#| tbl-cap: "Cycle threshold values"
#| fig-pos: "H"

table_data2 <- data.frame(
  Sample = c("FP6 R w0", "FP6 R w2pre", "FP6 R w0", "FP6 R w2pre", "FP6 R w0", "FP6 R w2pre"),
  Target = c("myhc1", "myhc1", "mhc2a", "mhc2a", "myhc2x", "myhc2x"),
  Ct1 = c(19.798, 18.944, 21.029, 19.549, 27.019, 24.871),
  Ct2 = c(19.901, 19.240, 21.247, 19.304, 26.898, 24.105),
  Ct3 = c(19.621, 19.861, 20.627, 19.580, 25.907, 24.256),
  Avg. = c(19.7733333, 19.3483333, 20.968, 19.478, 26.6079534, 24.4106166),
  Ref. = c("b2m", "b2m", "b2m", "b2m", "b2m", "b2m"),
  Ref_Ct1 = c(24.670, 22.913, 24.670, 22.913, 24.670, 22.913),
  Ref_Ct2 = c(24.513, 23.950, 24.513, 23.950, 24.513, 23.950),
  Ref_Ct3 = c(24.691, 23.819, 24.691, 23.819, 24.691, 23.819),
  Ref_Avg = c(24.625, 23.561, 24.625, 23.561, 24.625, 23.561),
  ΔCt = c(-4.851, -4.212, -3.657, -4.083, 1.983, 0.850),
  TwoΔΔCt = c(28.867, 18.536, 12.613, 16.944, 0.253, 0.555))



table_data2 %>% gt() %>% tab_source_note(
    source_note = "Number of cycles to reach the cycle threshold (CT) has changed from week 0 to week 2. The number of cycles decreased for myhc1 from an average of 19.7 to 19.3 cycles. For myhc2a, the cycles dropped from 20.9 to 19.4, and for myhc2x it went from 26.6 to 24.4 cycles. This indicates that the gene expression increased from week 0 to week 2 as fewer cycles indicate higher gene expression.") %>%
  tab_options(
    table.font.size = px(9), 
    table.width = px(500))


```

```{r}
#| message: false
#| echo: false
#| warning: false
#| label: tbl-x2
#| tbl-cap: "Percentage amount of gene expression for the different muscle types"


table_data3 <- data.frame(
  `100%` = c(41.733, 36.034),
  myhc1 = c("69.17 %", "51.44 %"),
  myhc2a = c("30.22 %", "47.02 %"),
  mhc2x = c("0.61 %", "1.54 %"))

colnames(table_data3) <- c("100%", "myhc1", "myhc2a", "myhc2x") 

table_data3 %>% gt() %>% tab_source_note(
  source_note = "The amount of gene expression for the different muscle fiber types has changed from week 0 to week 2. The expression of Muscle Fiber Type 1 (myhc1) and Muscle Fiber Type 2a (myhc2a) decreased, while Muscle Fiber Type 2x (myhc2x) increased.") %>%
  tab_options(
    table.font.size = px(9), 
    table.width = px(500)    
  )


```

## Discussion

### Dilution Series

From our dilution series we saw that as the dilution increases, Ct values rises, indicating that the cDNA concentration decreases which is to be expected in a properly diluted sample series [@svec2015]. However, Ct values are slightly inconsistent for higher dilutions, indicated with “Undet.” in @tbl-x. This suggests potential variability in detecting low cDNA amounts. Furthermore, we calculated the slope of the standard curve by plotting the logarithm of template concentration against the Ct values and got a slope of -2.6104. The theoretical ideal slope is -3.33 which corresponds to 100% primer efficiency [@svec2015]. This further complicated our calculated primer efficiency as 141.59% is far above the acceptable range of 90-110% [@svec2015]. Our findings indicate issues within our experiment, suggesting that the primer did not efficiently amplify the target during each PCR cycle as intended. This may have skewed the relative quantification of gene expression and lead to false interpretation of our results.

The results shown in @tbl-x1 highlight important aspects of reproducibility and variability at different sample dilutions. As expected, the average Ct values increase with dilution since lower concentrations require more cycles to reach the detection threshold [@Livak]. The standard deviation (SD) values are increasing as the sample is diluted. This indicates that as the template concentration decreases, the variability in the amplification increases. Lastly*,* the coefficient of variation (CV) is relatively low for the undiluted sample and for the ½ dilution, reflecting minimal variability and high precision in these concentrations. However, the CV value rises for the 1/10 dilution, suggesting increased variability in the measurements as the sample becomes more diluted. The increased SD and CV values at higher dilutions could be indicative of challenges such as pipetting errors or reduced efficiency in template detection.

### Gene Expression

We compared the myosin heavy chain gene expression level for pre- (w0) and post-intervention (w2pre) and found an increased expression of myhc 1 and 2a. Both genes show higher expression at 2 weeks (w2pre) compared to timepoint 0 (w0). The relative expression of these genes increases as indicated by the ΔCt and 2\^ΔΔCt values (@tbl-ct). Although myhc 2x also shows some increase in expression after 2 weeks, the magnitude of the increase is smaller compared to myhc 1 and 2a. This suggests a more gradual or less pronounced activation of myhc 2x in response to the training intervention. Overall, our findings suggest that strength training induces changes in gene expression, with myhc 1 and 2a showing more pronounced changes compared to myhc 2x.

There are limited studies who explain the changes in myosin heavy chain (myhc) gene expression after two weeks of strength training. Wilborn and Willoughby stated that after 8 weeks of heavy strength training, gene expression for myhc 1 and 2a increased, while that for mych 2x decreased [@wilborn2004]. We believe that the observed changes would correspond with those of Wilborn and Willoughby, but to a lesser extent. Their findings align with our findings, as we observed an increase in gene expression for both myhc 1 and myhc 2a, but it contradicts our observation of an increase in myhc 2x. The increase in myhc2x, associated with the fastest and most explosive muscle fibers, is unexpected, as this fiber type is typically reduced with prolonged training in favor of more endurance- or strength-oriented fibers (Type I and IIa) [@Pette2000]. This might indicate that our results represent an initial upregulation phase where all isoforms (myhc 1, 2a, 2x) are expressed to adapt to increased training stress. A longer training duration, such as that of Wilborn and Willoughby’s 8-week period, might show a clearer downregulation of myhc 2x as type 1 and 2a fibers are prioritized for strength and endurance adoptions [@Pette2000].However, it is also possible that this reflects technical errors, such as high primer efficiency or biological variation.

### Deviation

Our findings suggest that technical issues or inherent flaws in our primer design may have led to inaccurate results or misinterpretation of the data. A possible cause of deviation may be cross-contamination in reagents or samples which may lead to higher-than-expected amplifications. Our primer design might have been poor causing the primer to bind to unintended sequences or to themselves. Lastly, poor pipetting might have led to incorrect volumes during dilution preparation resulting in inaccurate standard curves. To address these issues, we could repeat the experiment with a new dilution series where we ensured careful pipetting to avoid errors. Additionally, negative controls could be used to confirm the absence of contamination in reagents or samples.

## Conclusion 

In conclusion, the results from our qPCR experiment provided valuable insights into the changes in gene expression related to muscle fiber types after two weeks of strength training. While our findings suggest an increase in gene expression for myhc 1, 2a, and 2x, we observed potential technical challenges that may have affected the accuracy of our results. Despite these limitations, our results align with previous studies on myosin heavy chain gene expression, indicating a likely up-regulation of myhc 1 and 2a with less pronounced changes for myhc 2x. Although a longer duration of training may provide further clarification on the dynamics of myosin heavy chain gene expression in response to strength training. Future experiments should address technical concerns, including pipetting errors and contamination, to get more reliable results.
